src/sbt-test/sbt-dao-generator/postgres/src/main/resources/db/migration/V1__Create_Tables.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE dept (
+  DEPT_ID    BIGINT NOT NULL,
+  DEPT_NAME  VARCHAR(20),
+  VERSION_NO INTEGER,
+  PRIMARY KEY(DEPT_ID)
+);
+
+CREATE TABLE emp (
+  EMP_ID     INTEGER NOT NULL,
```

Comment:
[IMO] dept か emp のどちらかで SERIAL 使っておきたいですね。
```suggestion
  EMP_ID     SERIAL,
```
は如何でしょう？